Source/ARTDataQuery.m

```diff
@@ -54,7 +54,7 @@ - (NSMutableArray *)asQueryItems {
     NSMutableArray *items = [super asQueryItems];
     if (self.untilAttach) {
         NSAssert(self.realtimeChannel, @"ARTRealtimeHistoryQuery used from outside ARTRealtimeChannel.history");
-        if (self.realtimeChannel.state != ARTRealtimeChannelAttached) {
+        if (self.realtimeChannel.state != ARTChannelStateAttached) {
```

Comment:
On a separate note, this change seems quite big (changing the naming everywhere) yet you have no comments in the commit as to why you did this? Use Git to tell a story if you can.